### ISSUE TITLE
Automate Flocking's unit tests across browsers with testem and the GPII Quality Infrastructure

### ISSUE THREAD

**Issue by colinbdclark** [colinbdclark/Flocking#176]
Testem provides a very convenient way of running unit tests automatically across all browsers. I've started using it for [osc.js](https://github.com/colinbdclark/osc.js) and it's been very useful.

Once cross-browser test automation is in place, we should also use one of the Vagrant VMs provided by the [GPII Quality Infrastructure](https://github.com/GPII/qi-development-environments) so that we can eventually have access to solid cross-platform continuous integration.

**Comment by colinbdclark** [comment]
Testem support was added to Flocking at 1aec8fb540b246c03dabfaeb35ff2e8daf7b32c0. Use of the QI's Linux Vagrant VM is also available, however it will be good to update this with a Windows VM instead at some point. However, I'm closing this and will file a separate ticket about this.
